aiokafka/record/_crecords/legacy_records.pyi

```diff
@@ -0,0 +1,70 @@
+from typing import Any, Generator, final
+
+from typing_extensions import Literal, Never
+
+from aiokafka.record._protocols import (
+    LegacyRecordBatchBuilderProtocol,
+    LegacyRecordBatchProtocol,
+    LegacyRecordMetadataProtocol,
+    LegacyRecordProtocol,
+)
+
+@final
+class LegacyRecord(LegacyRecordProtocol):
+    offset: int
+    attributes: int
+    key: bytes | None
+    value: bytes | None
+    crc: int
+    def __init__(
+        self,
+        offset: int,
+        timestamp: int,
+        attributes: int,
+        key: bytes | None,
+        value: bytes | None,
+        crc: int,
+    ) -> None: ...
+    @property
+    def headers(self) -> list[Never]: ...
```

Comment:
Why `Never`?

Reply:
Sequence[Never] is zero-length sequence

Reply:
But this is incorrect type for `headers` property

Reply:
Why? Headers in LegacyRecord are always empty.

Reply:
As I understand, the idea is to have common interface in both kinds of records. And you use different signatures for them, even though compatible. Ok, let's keep it this way, it's easy to fix anytime.